sub/source/index.rst

```diff
@@ -24,6 +24,7 @@ Getting more info
 .. toctree::
    :hidden:
    
+   Choosing a Ground Control Stations <docs/common-choosing-a-ground-station>
```

Comment:
```suggestion
   Choosing a Control Station Software <docs/common-choosing-a-ground-station>
```
As I understand it the actual control station is the hardware, whereas this page is about choosing software to run on the control station hardware, so if it's getting included then I think this label is more intuitive.

Reply:
we use GCS to refer to the hardware/software combination...